File: python/7 kyu/321.py
# https://www.codewars.com/kata/609eee71109f860006c377d1/train/python

# Passed

def last_survivor(letters, coords):
    arr = list(letters)
    for idx in coords:
        del arr[idx]
        
    result = "".join(arr)
    return result

output = last_survivor('zbk', [2, 1])
print(output)

File: nodejs/8 kyu/321.js
// https://www.codewars.com/kata/55a75e2d0803fea18f00009d/train/javascript

// Passed

function slope(points) {
  let [x1, y1, x2, y2] = points;
  let slopeValue = x2 - x1 !== 0 ? String((y2 - y1) / (x2 - x1)) : "undefined";
  return slopeValue;
}

const output = slope([10, 50, 30, 150]);
console.log(output);